src/module.c

```diff
@@ -6498,7 +6629,9 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
 cleanup:
     if (reply) autoMemoryAdd(ctx,REDISMODULE_AM_REPLY,reply);
     if (ctx->module) ctx->module->in_call--;
-    if (c) moduleReleaseTempClient(c);
+    if (c) {
+        moduleReleaseTempClient(c);
+    }
```

Comment:
why did you do that?

Reply:
artifact of other code changes be adding then removed.  will revert